packages/compass/src/app/styles/deprecated-bootstrap-styles/globals.less

```diff
@@ -2,9 +2,6 @@
 // Compass. These styles are globals from bootstrap which we have removed
 // from the global namespace and are isolated to this plugin.
 .with-global-bootstrap-styles {
-  line-height: 1.428571429; // 20/14
```

Comment:
Just an idea, this seems to be the same base line-height/font size as LeafyGreen's, maybe we don't need to remove it, we could just move it here: https://github.com/mongodb-js/compass/blob/abddc77eafe958e59e0fc393b4fb9e937bd4374e/packages/compass-home/src/components/home.tsx#L396

Another idea would be to wrap the entire `<Home>` within a `<Body as="div">`